app/src/main/java/com/example/KnowYouLimit/dto/drinks.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.KnowYouLimit.dto
+
+data class drinks(var name: String, var category: String, var abv: String, var size: String) {
+
+    override fun toString(): String{
+        return "$name $category $abv $size"
+    }
+}
```

Comment:
This addition was for the drink search DTO which essentially is what will be shown when you go to search for a drink. This allows for you to test if data is being brought in and accessible when you implement retrofit down the road. If you need help with retrofit I would have done it here but did not know the BASE_URL and link to the JSON data.